tests/test_virtualenv.py

```diff
@@ -483,6 +483,24 @@ def test_reuse_conda_environment(make_one):
     assert reused
 
 
+@has_conda
+def test_micromamba_environment(make_one, monkeypatch):
+    conda_path = shutil.which("conda")
+    which = shutil.which
+    monkeypatch.setattr(
```

Comment:
Are we doing this so you can run the test if you don't have micromamba installed?

Reply:
Yes. It's easy to install but this is even easier to get proper coverage.

(Might be nice for more of the tests to just check the commands issued instead of requiring installs)

Reply:
Thanks for clarifying

Reply:
Added a comment.